packages/control-property-editor/src/components/RTAToolBar.tsx

```diff
@@ -0,0 +1,65 @@
+import { UIDefaultButton, UIIconButton, UILabel, UiIcons } from '@sap-ux/ui-components';
+import { ReactElement, useEffect } from 'react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { appMode, rtaEvent } from '@sap-ux-private/control-property-editor-common';
+import type { RootState } from '../store';
+import { ChangesSlice } from '../slice';
+
+export function RTAToolBar(props: any): ReactElement {
```

Comment:
I wouldn't use the term `RTA` in the CPE, because it would just be confusing as it is already a UI5 component and it would not be clear if we talk about the UI5 part or CPE.

I don't think we need any specific name for the toolbar or the events.